rubygem-kafo_parsers/rubygem-kafo_parsers.spec

```diff
@@ -50,10 +50,6 @@ gem install --local --install-dir .%{gem_dir} \
             --force %{SOURCE0} --no-rdoc --no-ri
 %{?scl:"}
 
-%build
```

Comment:
should not be needed thanks to https://github.com/theforeman/kafo_parsers/commit/206903a28018a2926a67da7bfc61d0f6dba74657